content/en/docs/How To/sharding.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Sharding"
+date: 2020-10-29
+weight: 410
+description: >
+  Sharding how-to
+---
+
+Sharding is enabled in a topology by specifying a `sharding` field in the TOML
+`[output]` section. That field must be shardable though, in other words, a 
+sharding function must be associated to that field. Let's see how that works.
+
+
+#### The baker.Components structure
+
+The `baker.Components` structure links elements that may appear in the 
+configuration to the code eventually running when these elements are used
+inside a topology.
+
+
+#### Register sharding functions
+
+Sharding functions that may be used in topologies are stored inside of 
+the `ShardingFuncs` field of [`baker.Components`](https://pkg.go.dev/github.com/AdRoll/baker#Components).
+
+```go
+ShardingFuncs map[baker.FieldIndex]ShardingFunc
+```
+
+And a [`ShardingFunc`](https://pkg.go.dev/github.com/AdRoll/baker#ShardingFunc)
+is a hash function that returns an `uint64` `for baker.Record`
+
+```go
+type ShardingFunc func(Record) uint64
+```
+
+Finally, filling `ShardingFuncs` is a matter of associating a shardable field to
+the sharding function that implement the hashing of that field.
+
+
+#### Putting it all together
+
+The following is an example of an hypothetical record schema with 3 fields 
+named `timestamp`, `city` and `country`. Let's say that we'd like to use 
+`timestamp` and `country` for sharding but not `city`. This is how implementing
```

Comment:
As in a previous comments, as we can only select 1 field at a time for sharding, maybe I'dd add here that info, also saying that the system provides multiple functions so that the user can choose one or the other at his will